src/security-rules/security-rules-api-client.ts

```diff
@@ -24,8 +24,8 @@ const RULES_V1_API = 'https://firebaserules.googleapis.com/v1';
 export interface Release {
   readonly name: string;
   readonly rulesetName: string;
-  readonly createTime: string;
-  readonly updateTime: string;
+  readonly createTime?: string;
+  readonly updateTime?: string;
```

Comment:
Is there a reason to expect that we won't get the `create` and `updateTime`s back, or you're just signaling that we don't rely on them for anything?

Reply:
I use the same interface as an input argument when updating/patching a release. In this case I wanted the flexibility to not specify these 2 attributes. And you're right -- we also don't rely on them for anything else.